app/controllers/spree/admin/payments_controller.rb

```diff
@@ -171,12 +171,8 @@ def load_payment
       end
 
       def authorize_stripe_sca_payment
-        return unless @payment.payment_method.class == Spree::Gateway::StripeSCA
```

Comment:
it turns out backoffice payments only allow Cash and StripeSCA, so we better validate that as early as possible. Regardless of the authorization, etc.

Reply:
It can show up if the enterprise has it configured but nothing is stopping you from using it :shrug: 

![Screenshot from 2021-06-21 11-47-52](https://user-images.githubusercontent.com/762088/122742816-a0ca5f80-d286-11eb-9534-d7211e080de7.png)
![Screenshot from 2021-06-21 11-48-20](https://user-images.githubusercontent.com/762088/122742824-a4f67d00-d286-11eb-8ef7-c723fc284d57.png)

IMO this needs to be reconsider but it falls out of the scope of this PR.